front/src/styles/scss/applications/stdcmV2/_card.scss

```diff
@@ -61,39 +61,101 @@
   }
 
   .stdcm-v2-card__body {
-    padding: 0.688rem 1rem 0.8125rem 1.5rem;
-    // TODO: remove this style after ui-core component is implemented
-    .suggestions * label,
-    .stdcm-v2-via-stop-for * label {
+    padding: 14px 24px 16px 8px;
+
+    .location-line {
+      padding-left: 16px;
+      display: flex;
+
+      .ci-input {
+        padding: 0 10px 0 0 ;
+      }
+    }
+
+    // Fix osrd-ui styles
+    label {
       margin-bottom: 0;
     }
 
     .feed-back {
-      padding-right: 0.3rem;
-      padding-left: 0;
+      padding: 0;
+    }
+
+    .date-picker {
+      .feed-back {
+        padding-right: 10px;
+        padding-left: 16px;
```

Comment:
```suggestion
```

Reply:
I need to keep this padding-left, it's used to display correctly the datePicker error :/ That's why I had to keep the `padding-left: 16px` everywhere